examples/bind-and-convert-jsx/src/app.jsx

```diff
@@ -11,7 +11,7 @@ contentView.append(
   </Stack>
 );
 
-/** @param {CheckBoxSelectEvent} ev */
+/** @param {import('tabris').CheckBoxSelectEvent} ev */
```

Comment:
can you use the global variable instead of an import like in examples/bind-and-convert-jsx/src/ExampleComponent.jsx?